tests/integration/mounts_nodev_nosuid_noexec.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
```

Comment:
I'd rather have this in `mounts_sshfs.bats`, as a separate test case.

You can move the fuse mount setup phase to a separate function and call it with different parameters.

Reply:
Should be addressed by 0614fdb40f72ac54ccc6f82f66ecc80f2f9e5747 / b6b18dd6672cbc0fb7668db20df62c44c0672790.